tests/ui/crate_level_checks/entrypoint_recursion.stderr

```diff
@@ -1,11 +0,0 @@
-error: recursing into entrypoint `a`
```

Comment:
this seems like an unintentional deletion

Reply:
I don't think this has worked at all, the other test in the same file doesn't have a `.stderr` file either. I'm guessing compiletest-rs didn't care that there was an extraneous file lying around if the test passes without output?

Reply:
ah, seems fair. ty!